validation/cross_validate_rust_python.py

```
@@ -100,13 +113,13 @@ def carreau_yasuda_python(shear_rate):
-    print(f"  Method: apparent_viscosity(shear_rate)")
-    
-    # Try to test if we can create a blood model
-    # Note: This depends on cfd_python API structure
-    print(f"\n  TODO: Add cfd_python API test if blood model is exposed")
+    blood = cfd_python.CarreauYasudaBlood()
+    for gamma_dot in test_shear_rates:
+        mu_python = carreau_yasuda_python(gamma_dot)
+        mu_rust = blood.apparent_viscosity(gamma_dot)
+        error = abs(mu_python - mu_rust) / mu_python * 100
+        assert error < 0.01, f"Viscosity mismatch at shear rate {gamma_dot}: {error:.4f}% error"
+    print(f"  Formula matches Python implementation ✓")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)